packages/mermaid/src/schemas/config.schema.yaml

```
@@ -2037,8 +2037,6 @@ $defs: # JSON Schema definition (maybe we should move these to a separate file)
     type: object
     unevaluatedProperties: false
     properties:
-      useMaxWidth:
-        default: false
       padding:
         default: 8
```

nitpick: we should give this a `type: number`.

```suggestion
        type: number
        default: 8
```